Python/initconfig.c

```diff
@@ -1050,7 +1050,7 @@ _PyConfig_AsDict(const PyConfig *config)
 static PyObject*
 config_dict_get(PyObject *dict, const char *name)
 {
-    PyObject *item = PyDict_GetItemString(dict, name);
+    PyObject *item = PyMapping_GetItemString(dict, name);
```

Comment:
PyDict_GetItemString returns a borrowed reference, PyMapping_GetItemString returns an owned reference.

Reply:
Thanks for reminding, it has been replaced by `PyMapping_GetItemString`.